lib-model/lib-model-consume/src/main/java/com/uber/nullaway/libmodel/StubxFileWriter.java

```diff
@@ -0,0 +1,122 @@
+package com.uber.nullaway.libmodel;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/** Simple writer for the astubx format. */
+final class StubxFileWriter {
```

Comment:
As we talked about, we should remove the duplicate copy of this class in the jarinfer code and have jarinfer depend on this module